app.py

```diff
@@ -21,7 +21,10 @@
 
 def get_chromecast():
     """Returns a connection to the Chromecast."""
-    return pychromecast.Chromecast(app.config['CHROMECAST_IP'])
+    try:
+        return pychromecast.Chromecast(app.config['CHROMECAST_IP'])
+    except pychromecast.ChromecastConnectionError:
+        pass
```

Comment:
Once the logging PR is pushed add logging here so that we can see it in the logs. Also will help when determining on frequently this fails.
`self.log.error("Unable to connect to Chromecast (%s)" %app.config['CHROMECAST_IP'])`